uktrade/data-hub-frontend
# Duplicate contact warning

## Description of change

Adds a warning message when a user is about to create a new contact with an email already used in some other contact.

## Test instructions

1. Go to a company page of your choice
2. Click on the _Company contacts_ tab
3. Click on the _Add contact_ button
4. Fill out the form with a unique _email_
5. Submit the form
6. The form should redirect you to the details page of the newly created contact
7. Repeat steps 1 to 5, except in step 4 don't use a unique _email_, but use the same email as before
8. After the form was submitted there should appear a duplicate email message in a blue box on top of the form
9. Not heeding the warning, submit the form again
10. The form should redirect you to the details page of the newly created contact with a duplicate email
11. This flow should also work 

## Screenshots

<img width="699" alt="Screen Shot 2021-08-13 at 4 59 17 PM" src="https://user-images.githubusercontent.com/2333157/129386798-3500bbde-39b1-4fe2-ab88-b8acf23fd229.png">
